### DISCUSSION TITLE
Disable or customize date parsing for suggested citation of my own work?

### DISCUSSION THREAD

**Discussion by maptv** [quarto-dev/quarto-cli#7748] [Q&A] [answered]
### Description

I would like to be able to show a citation in my output as per https://quarto.org/docs/authoring/create-citeable-articles.html with a custom format, e.g. today's date: 2023+274.
Is there a way to disable or customize date parsing by Pandoc/Quarto?
Either passing a literal date that is included verbatim or writing code to format the date would work for me.
I use my date format in the listing page and in every blog post of my site: https://maptv.github.io but the citeable articles feature does not seem to respect pandoc lua filters or the `date-format` property.

**Comment by mcanouil** [comment, marked as answer]
The styling of citations is controlled by the citation style (CSL) mentioned in the pages you linked.
Choose the one that fit your needs and/or modify one.

Side note: use `include-` options to include script/style and set those in `_quarto.yml` to share them across your website, instead of copying th code and having to change the path to your JavaScript.

**Reply by maptv** [reply]
Thank you for the great advice. CSL is definitely the best way to customize dates, but I think it is too inflexible for my very unique needs. I moved all of my script/style chunks to HTML files, so I could include them with `include-after-body`, as you suggested. I then created a new HTML file with a script tag that modifies the dates in the bibtex and cite-as citations. Thanks again for your help!
